modules/virt-editing-vm-web.adoc

```diff
@@ -14,31 +14,40 @@
 ifeval::["{context}" == "virt-editing-vm-template"]
 :virt-vm-template:
 :object: virtual machine template
-:object-gui: Virtual Machines
-:object-vm-overview: VM Template Details
+:object-gui: Templates
+:object-vm-overview: Virtual Machine Overview
+:object-vm-details: VM Template Details
 endif::[]
 
 ifeval::["{context}" == "virt-edit-vms"]
 :virt-vm:
 :object: virtual machine
 :object-gui: Virtual Machines
 :object-vm-overview: Virtual Machine Overview
+:object-vm-details: Details
+:object-details: Details
```

Comment:
This conditional is only defined for the vm context and not the template context. It is never used in the reusable procedure content.

Reply:
Yes, so here is the edited set of conditionals:

ifeval::["{context}" == "virt-editing-vm-template"]
:virt-vm-template:
:object: virtual machine template
:object-gui: Templates
:object-vm-details: VM Template Details
endif::[]

ifeval::["{context}" == "virt-edit-vms"]
:virt-vm:
:object: virtual machine
:object-gui: Virtual Machines
:object-vm-details: Details
endif::[]